pkg/master/master.go

```diff
@@ -372,9 +372,11 @@ func (m *Master) InstallLegacyAPI(c *completedConfig, restOptionsGetter generic.
 	}
 
 	if c.ExtraConfig.EnableCoreControllers {
+		controllerName := "bootstrap-controller"
```

Comment:
`bootstrapController.Name` would be better.

Reply:
Done